packages/web3-providers-ws/src/index.js

```diff
@@ -20,7 +20,7 @@
  * @date 2017
  */
 
-"use strict";
+'use strict';
```

Comment:
Mentioned this in #3180. What do you think about keeping the unrelated style changes out? It reduces the diff clarity quite a bit and makes it harder to see what's critical. 

How the project is linted/styled could be addressed later at some point as separate effort, with a linter, a way to enforce the standard (in CI or a pre-commit hook like [husky](https://www.npmjs.com/package/husky#husky)).  And a strategy to introduce the changes in a uniform way.

Reply:
Thanks for the hint! I will keep those changes in this PR and will not change the code-style in all newly opened PRs.